lua/telescope/pickers.lua

```diff
@@ -1143,13 +1143,6 @@ function Picker:get_result_processor(find_id, prompt, status_updater)
     end
 
     self.sorter:score(prompt, entry, cb_add, cb_filter)
-
-    -- Only on the first addition do we want to set the selection.
-    -- This allows us to handle moving the cursor to the bottom or top of the window
-    -- depending on the strategy.
-    if count == 1 then
-      self:_do_selection(prompt)
-    end
```

Comment:
@tjdevries are you trolling me. Laughing. That just forces selection_strategy `reset`, breaking `row`, ...

If we have reset set we will move to the bottom | top regardless. So why is it here? Everything seems to work without it.

Reply:
LOL, just saw this. Who knows. This is a confusing landscape of confusion.

Reply:
:rofl: guess it wasn't important